process/transactions/tokens.go

```diff
@@ -0,0 +1,240 @@
+package transactions
+
+import (
+	"encoding/hex"
+	"time"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/parsers"
+)
+
+const (
+	metachainESDTAddress = "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u"
```

Comment:
no, please use the elrond-go/vm/address.go -> ESDTSCAddress value

Reply:
done.